myuw/dao/term.py

```diff
@@ -391,6 +403,10 @@ def add_term_data_to_context(request, context):
 
     context['first_day'] = cur_term.first_day_quarter
     context['last_day'] = cur_term.last_day_instruction
+    context["first_day_quarter"] = cur_term.first_day_quarter
+    context["last_day_instruction"] = cur_term.last_day_instruction
```

Comment:
Just asked Matt Winslow why final week starts on the Saturday. here is his response: "Yes, we have ‘common finals’ on Saturday. These are finals for classes that have multiple sections across multiple times but opt in to have a single ‘common’ final. We hold those on Saturdays."